attributes/default.rb

```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```

Comment:
we can fix it and then the users will be able to override , but the problem is that when the users will use Deb the system will not have values for  node['lsb']['codename']  , i checked it with chef-shell  so in this case the cookbook will fail...